de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/simulator/JSSimulatorNodeModel.java

```diff
@@ -54,11 +48,15 @@
     extends AbstractWizardNodeModel<JSSimulatorViewRepresentation, JSSimulatorViewValue>
     implements PortObjectHolder {
 
+  private final JSSimulatorConfig m_config = new JSSimulatorConfig();

```

Comment:
We should have class variables together at the top before any methods. *port* should be defined right after m_config. https://www.cs.cornell.edu/courses/JavaAndDS/JavaStyle.html#fielddecs